content/administration/maintain/update.rst

```diff
@@ -19,7 +19,7 @@ definitions:
 
 Updating (an Odoo installation)
   Refers to the process of obtaining the latest revision of the source code for
-  your current Odoo Edition. For example, updating your Odoo Enterprise 13.0 to the
+  your current Odoo Edition. For example, updating your Odoo Enterprise {CURRENT_RELEASE} to the
   latest revision.
```

Comment:
this makes less sense if current_release is also latest_release

```suggestion
  your current Odoo Edition. For example, updating Odoo Enterprise {PREVIOUS_RELEASE} to {CURRENT_RELEASE}, the
  latest revision.
```

Reply:
I reverted this change to leave 13.0.

Reply:
Actually, I removed LATEST_RELEASE and NEXT_RELEASE following this comment and https://github.com/odoo/documentation/pull/3051#discussion_r1035914356